src/schema/nodes/inline_footnote.ts

```diff
@@ -31,7 +30,6 @@ export const inlineFootnote: NodeSpec = {
   name: 'Footnote marker',
   attrs: {
     rids: { default: [] },
-    contents: { default: '' },
```

Comment:
We probably need to update the `jats-dom-parser` as well. I also wonder if we need to migrate older documents since i think we did store content there.

Reply:
Updated. The value of the attr should not be relevant, so we don't need to worry about older documents.

Reply:
Won't we need to remove the attribute as well from the node? I'm wondering what would happen if we don't.